### PR TITLE
Re enable all markers (except civ and quantity)

### PR DESCRIPTION
**Pull request by chris579**
**When merged this pull request will:**
- Re-enable the placement of the medical markers on the ingame map

Marking a medical FOB ingame would be quite nice so this could be re-enabled.

### CODE REVIEW

**Comment by jonpas** [comment]
Considering [this comment](https://github.com/chris579/ACE3/blob/ef41d66776b18a2859710a00c87459edb116fece/addons/map/CfgMarkers.hpp#L8-L12) I would say this entire `CfgMarkers` config should be removed, as there is a better way to place markers now.

**Comment by jonpas** [comment]
I guess everyone agrees with me.

**Comment by SilentSpike** [comment]
Yeah I agree 👍

**Comment by chris579** [comment]
Re enabled all markers except civ markers and quantity indicators.

**Comment by jonpas** [comment]
I think that's good. Is `Flag` hidden by default or was it enabled because it inherits from some other class?

**Comment by chris579** [comment]
I think `Flag` is hidden by default and `b_unknown` inherits from it.

**Comment by SilentSpike** [comment]
You can simplify the whole file:

```
class b_unknown;

// disable all civy markers (harbor etc.)
class c_unknown: b_unknown {scope = 1;};

// disable quantity indicators (fire team/squad/platoon ...)
class group_0: b_unknown {scope = 1;};
```

Though personally I don't see why the size and civilian markers should remain disabled. I'd just enable those too and delete this file.

**Comment by chris579** [comment]
Is there a reason why you should be able to create harbor markers while being ingame? But yes, we could delete the whole file.

**Comment by SilentSpike** [comment]
Looking at the vanilla config file now, I'm actually a little confused as there are no markers given `scope = 2`. Surely that would mean in vanilla you cannot place any markers? 😕

**Comment by chris579** [comment]
Afaik you're only able to place the hand drawn markers in vanilla so you have at least to set `scope = 2` to one of the base classes. Therefore deleting the whole config is not possible.

**Comment by jonpas** [comment]
> Afaik you're only able to place the hand drawn markers in vanilla

Which makes sense, given you only have a board marker and no flags on pins.

**Comment by chris579** [comment]
Anyway, is there any reason why somebody should be able to place harbor or hospital markers? I think this would mess up the marker type selection when trying to quickly place a marker.

**Comment by jonpas** [comment]
I personally don't see why, I'd only leave markers that look hand-drawn.

File: addons/map/CfgMarkers.hpp [hunk 1]
```diff
@@ -4,36 +4,9 @@ class CfgMarkers {
...
     class n_unknown: b_unknown {};
-    class n_hq: n_unknown {scope = 1;};
-    class n_installation: n_unknown {scope = 1;};
-    class u_installation: n_unknown {scope = 1;}; // i have no idea...
-    class n_maint: n_unknown {scope = 1;};
-    class n_med: n_unknown {scope = 1;};
-    class n_service: n_unknown {scope = 1;};
-    class n_support: n_unknown {scope = 1;};
-
     class o_unknown: b_unknown {};
```
Comment (SilentSpike) [inline addons/map/CfgMarkers.hpp]: This line can also be deleted (it inherits from `b_unknown` already)

File: addons/map/CfgMarkers.hpp [hunk 2]
```diff
@@ -4,36 +4,9 @@ class CfgMarkers {
...
-    // important markers here.
-    // Keep in mind that all of these can still be placed in the editor.
-    class b_hq: b_unknown {scope = 1;};
-    class b_installation: b_unknown {scope = 1;};
-    class b_maint: b_unknown {scope = 1;};
-    class b_med: b_unknown {scope = 1;};
-    class b_service: b_unknown {scope = 1;};
-    class b_support: b_unknown {scope = 1;};
-
     class n_unknown: b_unknown {};
```
Comment (SilentSpike) [inline addons/map/CfgMarkers.hpp]: This line can also be deleted (it inherits from `b_unknown` already)

**Comment by jonpas** [comment]
@SilentSpike All good for you?

**Pull request merged**